hack/update-codegen.sh

```diff
@@ -64,7 +64,7 @@ ${CODEGEN_PKG}/generate-groups.sh "deepcopy" \
   --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt
 
 # Depends on generate-groups.sh to install bin/deepcopy-gen
```

Comment:
Ah, now that I understand the requirement, you aren't using the vendored version here AFAICT, but whatever happens to be in `$GOPATH/src/k8s.io`, at least as far as I can understand from reading `go help packages`

It think should be `go run $(dirname $0)/../vendor/k8s.io/code-generator/cmd/deepcopy-gen`

The other instance happens to be okay because this is knative.dev/pkg... but you may want to change it to be consistent

Reply:
I'm not sure this is true. I tried removing code-generator from my GOPATH and the script succeeded without pulling code-generator back into GOPATH. Since the repo is now using go modules so GOPATH should not be used, and since the go.mod version is >= 1.14 go will default to running with -mod=vendor. Therefore go run should run by default using vendored modules. I also tested this by vendoring a change to deepcopy gen and checking for the changed output.

Reply:
okay, I forgot go modules affects a lot of functionality of go itself. Glad it worked!

Reply:
vendor acts as GOPATH in go mod projects